### ISSUE TITLE
Accuracy without pre-training

### ISSUE THREAD

**Issue by 0316038** [seoungwugoh/RGMP#12]
Hi, I would like to ask about the ablation study without pre-training(-PT). 
How many epoch did you train and how to set the learning rate for the fine-tuning part to get the accuracy in Table 5? (J mean is 68.6 and F mean is 68.9)
Thanks for your attention and hope to hear your reply soon.

**Comment by seoungwugoh** [comment]
Hi, 
For the purpose of ablation study, we apply the same training schedule with the counter part (with pre-training).
Please refer to our paper for details.

**Comment by huhu5036** [comment]
Hi, Did you ever try to set the prior frame mask to all black?  I would like to know how much the reference frame and mask matter?  Thus, the method can really handle it well when the target appears again from disappeared.  
Thanks a lot,  and hope for your reply.

**Comment by seoungwugoh** [comment]
@huhu5036 We have the exact experiment in our paper (Fig.5 and Table.5). You can see the effect of the reference frame and the previous mask.
